templates/base.html

```diff
@@ -8,10 +8,13 @@
   </head>
   <body>
     <div class="mainbar">
-      <h1><a href="/" class="nolinkstyle">PolarBear</a></h1>
+      <h1><a href="/" class="nolinkstyle" style = 'padding-left: 10pt'>PolarBear</a></h1>
       {% if user and user.is_authenticated %}
       <div class="baroption">
         <p style = 'padding-right: 10pt'>welcome, {{ user.username }}</p>
+        <a href="/user/authors/{{ user.author.id }}" class="buttonlink">Profile</a>
```

Comment:
`/user/authors/{{ user.author.id }}` should be replaced with `{% url 'profile' user.author.pk %}`